string name
---
int32 error_code
string name
string parent
geometry_msgs/Pose pose
